javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -344,10 +347,33 @@ public static ClientMethodMapper getInstance() {
                             parameters);
                 }
 
+                String operationId;
+                if (operation.getOperationGroup() != null
+                        && operation.getOperationGroup().getLanguage() != null
+                        && operation.getOperationGroup().getLanguage().getDefault() != null
+                        && operation.getOperationGroup().getLanguage().getDefault().getName() != null
+                        && !operation.getOperationGroup().getLanguage().getDefault().getName().isEmpty()) {
+                    operationId = operation.getOperationGroup().getLanguage().getDefault().getName() + "_"
+                            + operation.getLanguage().getDefault().getName();
+                } else {
+                    operationId = operation.getLanguage().getDefault().getName();
+                }
+                JavaSettings.PollingDetails pollingDetails = settings.getPollingConfig(operationId);
```

Comment:
In a recent change, I actually added `operationId` to `ProxyMethod`. Maybe it can be reused.
https://github.com/Azure/autorest.java/blob/v4/javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java#L62-L75

Reply:
Awesome!